### ISSUE TITLE
⌨ Notifications Mark all as read action is not announced

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#10636]
## ⌨ Accessibility Issue

The notifications **Mark all as read** button does not trigger an announcement by screen readers of the action.

### WCAG 2.1 Criteria

[4.1.3: Status Messages (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html)

## 🦋 Expected Behaviour

When activating the Mark all as read button, the action should be announced by screen readers.

## 🕵️ Details

Add any additional details that could assist with troubleshooting/fixing the issue.

- **Operating System**: macOS 12.7.5
- **Browser**: Safari Version 17.5
- **Assistive Technology**: VoiceOver

## 📋 Steps to Reproduce

1. View notification on nav and activate link
2. Mark all as read
3. Verify action not announced by VoiceOver

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] When activating the Mark all as read button, the action is announced by screen readers
